index.html

```diff
@@ -111,6 +111,49 @@ <h2>A project for first-time contributions</h2>
             <!-- make sure there is one line of space above and below your card -->
             <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+             <!-- ________  AROMA's card START ________  -->
+             <div class="card">
+                <p class="name">Aroma Gupta</p>
+                <p class="contact">
+                    <i class="fab fa-linkedin-in"></i>
+                    <a href="https://www.linkedin.com/in/aroma-g-429595203" target="_blank">@AromaGupta</a>
+                     <br>
```

Comment:
Hello @aroma-sketch, thank you for contributing. Everything looks great, just to keep uniformity across the cards, could you please remove this break tag in line 120. We would like all of your available social platforms on one line to keep the header the same height. After you make this change push back to this branch and we will get your card added to the project. Thank you and take care 😄